### PR TITLE
Update hand 6234 tip sensors

### PR DESCRIPTION
**Pull request by rnzenha-s**
## Proposed changes

Update tip sensors on hand

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_hand_config/6234/rates/sensor_data_polling.yaml [hunk 1]
```diff
@@ -50,5 +50,5 @@ ubi0_sensor_data_update_rate: {
 
 mst_sensor_data_update_rate: {
   TACTILE_SENSOR_TYPE_MST_MAGNETIC_INDUCTION: -1.,
-  TACTILE_SENSOR_TYPE_MST_TEMPERATURE: -1.
-  }
\ No newline at end of file
+  TACTILE_SENSOR_TYPE_MST_TEMPERATURE: 1.
```
Comment (dg-shadow) [inline sr_hand_config/6234/rates/sensor_data_polling.yaml:53]: What is this about? positive or negative integer rates but with decimal points? I guess they're all like this but can someone explain why? seems weird...

Comment (toliver) [reply sr_hand_config/6234/rates/sensor_data_polling.yaml:53]: We should add some comment explaining how this works.

- Positive numbers are frequency of polling in Hz
- -1 means "poll as frequently as possible". This normally means polling at 1Khz if there's only one type of data configured like this. Or polling at 500Hz if two types of data are configured as -1, en so on. That is, unless a positively conigured type of data needs to be polled instead. the "-1" data basically will fill all the gaps when nothing else has to be pollled.
- -2 means that this type of data is only polled once at initialization.

**Review by toliver — APPROVED** [review]

File: sr_hand_config/6234/rates/sensor_data_polling.yaml (hunk 1 above)
Comment (dg-shadow) [reply sr_hand_config/6234/rates/sensor_data_polling.yaml:53]: Ok, if this comment gets added I'm happy to pass.

Comment (dg-shadow) [reply sr_hand_config/6234/rates/sensor_data_polling.yaml:53]: (even though there are now two reviews. please add comments anyway.

Comment (rnzenha-s) [reply sr_hand_config/6234/rates/sensor_data_polling.yaml:53]: Created a separate PR for that:
https://github.com/shadow-robot/sr_hand_config/pull/269

**Pull request merged**